website/views/index.jade

```diff
@@ -4,11 +4,13 @@ html(ng-app="habitrpg", ng-controller="RootCtrl", ng-class='{"applying-action":a
   head
     title=env.t('titleIndex')
     // ?v=1 needed to force refresh
-    link(rel='shortcut icon', href='#{env.getBuildUrl("favicon.ico")}?v=4')
+    link(rel='shortcut icon', sizes="32x32", href='#{env.getBuildUrl("favicon.ico")}?v=4')
+    link(rel='shortcut icon', sizes="144x144", href='#{env.getBuildUrl("favicon_192x192.png")}?v=4')
 
     meta(charset='utf-8')
     meta(name='viewport', content='width=device-width, initial-scale=1.0')
     meta(name='apple-mobile-web-app-capable', content='yes')
+    meta(name='mobile-web-app-capable', content='yes')
```

Comment:
with that it should be able to pin habitica as "web app" to the android launcher , right?

Since we already have the ios version on it, we also could add this ....

ping @veeeeeee @lemoness